portfolio/src/main/webapp/script.js

```diff
@@ -188,4 +188,11 @@ function createNextButton(direction, isValid) {
   nextButton.className = isValid ? 'btn btn-secondary' : 'btn btn-secondary disabled';
   nextButton.innerHTML = direction == 'r' ? '&raquo;' : '&laquo;';
   return nextButton;
+}
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
nit: let's reformat so it's more readable
`new google.maps.Map(document.getElementById('map'), {
    center: {
        lat: 37.422,
        lng: -122.084
    },
    zoom: 16
 });`

Reply:
oops that didn't format how I intended lol hold on

Reply:
In this example: https://developers-dot-devsite-v2-prod.appspot.com/maps/documentation/javascript/examples/map-sync see how `center` and `zoom` are on their own lines? And the bracket opens on the line above `center`. And `new google.maps.Map(document.getElementById('map'), {` can all be on one line.